### ISSUE TITLE
Setup session submission

### ISSUE THREAD

**Issue by stephlocke** [lockedata/rcms#25]
## Detailed task
- Define any tracks / categories required
- Configure CFP details
- Add CFP content like FAQs
- Customise form

## Assessing the task
Try to perform the task. Use google and the system documentation to help - part of what we're trying to assess how easy it is for people to work out how to do tasks.

Use a 👍 (`:+1:`) reaction to this task if you were able to perform the task. Use a 👎 (`:-1:`) reaction to the task if you could not complete it. Add a reply with any comments or feedback.

## Extra Info
- Site: [odoo](//http://188.166.159.192:8069)
- System documentation: [odoo docs](https://www.odoo.com/page/docs)
- Role: Conference Admin
- Area: Setup

**Comment by Amymcdougall** [comment]
## Extra Info

Site: [odoo](http://188.166.159.192:8069)

**Comment by tuxette** [comment]
I haven't seen anything about paper submission in this system...

**Comment by Amymcdougall** [comment]
![image](https://user-images.githubusercontent.com/33873874/41029404-7e76f90e-6973-11e8-9365-6e70f6be6683.png)
![image 1](https://user-images.githubusercontent.com/33873874/41029448-93874b00-6973-11e8-8b00-91e1df04d5bc.png)

I believe this is how you do it and have shown Steph also.

**Comment by tuxette** [comment]
Yes but it does not really open something for people outside the admin team to submit proposals that will be reviewed later, does it? For me, it is made for creating a session in which only an admin can post a content.

**Comment by Amymcdougall** [comment]
But if you check out issue #37 I was able to submit a talk there and track it also. 

I think it's a little confusing to figure out but I'm pretty certain it all works? 😣🤞😂

**Comment by tuxette** [comment]
I went to odoo to try to understand (but I haven't been able to). Somehow, it's like you have registered to attend a session : 
![screenshot_2018-06-06_11-46-57](https://user-images.githubusercontent.com/4946478/41030847-89ae9a64-697f-11e8-9aeb-5a91848dacae.png)
It's all very confusing since the tasks are not made in a flow: can someone call for paper submission, someone else has access to this proposal and reviews/selects it and then put it in a track or an event (I don't really even know what is supposed to be a conference session)? I would say no because I am not able (as an admin) to see your proposal as in task #37 . But I'm unsure: this program is very very difficult for me to understand.
